app/pods/publications/publication/controller.js

```diff
@@ -276,10 +275,22 @@ export default class PublicationController extends Controller {
     } else {
       this.model.publicationFlow.status = status;
       this.publicationStatus = status;
+      if (status.isPublished) {
+        this.model.publicationFlow.closingDate = new Date();
+      } else {
+        this.model.publicationFlow.closingDate = null;
+      }
       this.model.publicationFlow.save();
     }
   }
 
+  @action
+  async setPublicationType(event) {
+    const publicationType = await this.store.findRecord('publication-type', event.id);
+    this.model.publicationFlow.set('type', publicationType);
+    this.model.publicationFlow.save();
+  }
+
```

Comment:
Deze code hoort hier niet, ik vermoed dat deze is teruggekomen door een merge conflict?
Deze code is verwijderd geweest in 
https://github.com/kanselarij-vlaanderen/kaleidos-frontend/commit/d421ce4f97f7710b7a3f8a5530c4dfd81c39adac#diff-8b8d309ae99f466d59cac502b38230ff09ff8e575b42f86d8d4aee02620f3adbR364

Dit is nu setPublicationMode, lijn 375-379

Reply:
Idd. Bedankt. Ik fix het meteen.

Reply:
Oei. Ik dacht dat dit over de sidebar ging :speak_no_evil: Wat verwarrend nu. Eens sidebar gemerged, zullen deze dingen (ongeveer :wink: ) vanzelf weggaan :-)

Reply:
Moet ik het aanpassen of heb jij het al gedaan XD

Reply:
Ik ga het nu fixen dus ^^